_posts/2020-09-25-strimzi-survey-results.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: post
+title:  "Insights from the first Strimzi survey"
+date: 2020-09-25
+author: tombentley
+---
+
+In August, with the help of the CNCF, we ran a survey of the Strimzi community. 
+This was prompted, in part, by the maintainers wanting to make decisions based on a better understanding of how the community is using Strimzi and what would best serve the common good in the future.
+As maintainers we spend a lot of time working with people for whom Strimzi isn't working, or who really want some particular feature. 
+We don't tend to hear much from people who can't or don't want to open issues or PRs or who don't interact on Slack.
+So this was an attempt to get an impression about how well Strimzi is surving _all_ of its community.
+
+If you took the time to complete the survey, thank you. 
+
+The rest this blog post is our analysis of the results of the survey.
+
+<!-- more -->
+
+First the numbers. 
+We had about 40 respondants overall, but since no questions were mandatory the numbers are different for each answer.
+We'd previously done an informal Twitter poll to answer the specific question: "how do you install Strimzi", this had more than twice as many votes. 
+Perhaps the difference is explained by the time investment required for a longer survey, or that we ran the survey in August when many people are taking vacation.
+
+Let's get on to the questions...
+
+## Usage
+
+Nearly 40% of respondants were using Strimzi in production.
+Over 50% were using it in a staging or development environment and
+about 15% were using it locally for development. 
+These don't sum to 100% because people could choose more than one option.
+
+## Kubernetes distributions and versions
+
+Over 55% of respondants were running Strimzi on OpenShift.
+This is likely a reflection of Red Hat being the company which started Strimzi.
+In comparison 25% were using it on EKS and the same on GKE.
```

Comment:
isn't it better to specify from which company EKS and GKE come? so Amazon and Google?